tests/baselines/reference/typeArgumentInferenceWithConstraintAsCommonRoot.errors.txt

```diff
@@ -11,7 +10,6 @@ tests/cases/compiler/typeArgumentInferenceWithConstraintAsCommonRoot.ts(7,1): er
     var g: Giraffe;
     var e: Elephant;
     f(g, e); // valid because both Giraffe and Elephant satisfy the constraint. T is Animal

```

Comment:
update comment. looks like it doesn't find a common supertype any more. is it supposed to?